samples/app-anonymous-users/csharp/ClientApp/src/components/shareview.jsx

```diff
@@ -19,11 +19,28 @@ const ShareView = () => {
     const [uShowCount, uShowSetCount] = useState(0);
 
     // Declare new state variables that are required to disable the submit vote button.
-    const [disabled, setDisabled] = useState(false)
+    const [isVoteBtnDisabled, setIsVoteBtnDisabled] = useState(false);
 
-    useEffect(() => {
-        microsoftTeams.initialize();
+    // Declare new state variables that are required to set the username of users who sign in using SSO too.
+    const [userName, setUserName] = useState("");
+
+    // Declare new state variables that are required to disable the Facebook button after login
+    const [IsdisabledfacebookButton, setIsdisabledfacebookButton] = useState(true);
```

Comment:
Can we rename this to `IsFacebookButtonDisabled` so it's easier to read.

I also think that current naming is the opposite of what it the variable does. When it's value is true I expect based on the naming that the button is disabled, when in fact it's the opposite.

Reply:
Fixed